app/controllers/notifications_controller.rb

```diff
@@ -3,7 +3,7 @@ class NotificationsController < AuthenticatedController
 
   def index
     notifications = current_user.notifications.newest.includes(
-      :actor, notifiable: [:user, :commentable]
+      :actor, :notifiable
```

Comment:
![image](https://user-images.githubusercontent.com/18374402/79198354-f080f000-7e65-11ea-8f30-e794914e04ca.png)

I removed this because it returns an error which states that the Card model does not have a user association. But i did some research after you asked this question and found a solution for it. https://stackoverflow.com/questions/42773318/eager-load-depending-on-type-of-association-in-ruby-on-rails